exporter/splunkhecexporter/factory.go

```diff
@@ -160,28 +183,40 @@ func createLogsExporter(
 	set exporter.CreateSettings,
 	config component.Config,
 ) (exporter exporter.Logs, err error) {
-	if config == nil {
-		return nil, errors.New("nil config")
-	}
 	cfg := config.(*Config)
+	if err = cfg.Validate(); err != nil {
+		return nil, err
+	}
```

Comment:
This is not needed, Validate is called implicitly by the config builder https://github.com/open-telemetry/opentelemetry-collector/blob/f663a7984e6bdff833435ee630346d824bd78252/component/config.go#L115